src/data/polynomial/algebra_map.lean

```diff
@@ -273,14 +285,20 @@ lemma aeval_eq_sum_range' [algebra R S] {p : R[X]} {n : ℕ} (hn : p.nat_degree
   aeval x p = ∑ i in finset.range n, p.coeff i • x ^ i :=
 by { simp_rw algebra.smul_def, exact eval₂_eq_sum_range' (algebra_map R S) hn x }
 
-lemma aeval_sum {ι : Type*} [algebra R S] (s : finset ι) (f : ι → R[X])
-  (g : S) : aeval g (∑ i in s, f i) = ∑ i in s, aeval g (f i) :=
-(polynomial.aeval g : R[X] →ₐ[_] _).map_sum f s
+lemma aeval_list_prod [algebra R S] (l : list R[X]) (x : S) :
+  aeval x l.prod = (l.map (aeval x)).prod :=
+map_list_prod (aeval x) _
+
+lemma aeval_multiset_prod [algebra R S] (s : multiset R[X]) (x : S) :
+  aeval x s.prod = (s.map (aeval x)).prod :=
+map_multiset_prod (aeval x) _
```

Comment:
@Vierkantor, can you give a second opinion on whether these lemmas are worth having? My feeling is that they're not, as the more general lemmas entirely cover them.

On the other hand, we already do this duplication for basic things like `aeval_zero`, so there is precedentm

Reply:
OK, I'll remove the lemmas about `aeval` in this PR and change the title.